tools/rpmuncompress.c

```diff
@@ -223,6 +223,7 @@ int main(int argc, char *argv[])
 
 	if (dryrun) {
 	    ec = EXIT_SUCCESS;
+	    free(cmd);
```

Comment:
Just move the existing free below the exit label so it gets handled for all cases now and in the future, I don't know why I've put the free where it is now. It's a fine example of why the clean-centrally-at-exit is so important in C.

Reply:
Oh and, thanks for spotting it!

Reply:
Well, the memory sanitizer barfing all over the console wasn't that hard to spot. Otherwise this would probably never been found.